### PR TITLE
Fix incorrect link in ComputeLogisticRegressionStandardDeviation class documentation

### LINKED ISSUE #3479: Incorrect link in ComputeLogisticRegressionStandardDeviation class documentation
It recommends using the implementation in Microsoft.ML.Mkl.Components, but it links to the page of the abstract class `ComputeLogisticRegressionStandardDeviation`.

### PR DESCRIPTION
**Pull request by codemzs**
fixes #3479

### CODE REVIEW

File: src/Microsoft.ML.StandardTrainers/Standard/LogisticRegression/LogisticRegression.cs [hunk 1]
```diff
@@ -477,8 +477,7 @@ internal static CommonOutputs.BinaryClassificationOutput TrainBinary(IHostEnviro
     /// <summary>
     /// Computes the standard deviation matrix of each of the non-zero training weights, needed to calculate further the standard deviation,
     /// p-value and z-Score.
-    /// If you need fast calculations, use the <see cref="ComputeLogisticRegressionStandardDeviation"/> implementation in the Microsoft.ML.Mkl.Components package,
-    /// which makes use of hardware acceleration.
+    /// If you need fast calculations, use this class' implementation in the Microsoft.ML.Mkl.Components package which uses Intel Math Kernel Library.
```
Comment (yaeldekel) [inline src/Microsoft.ML.StandardTrainers/Standard/LogisticRegression/LogisticRegression.cs]: >If you need fast calculations [](start = 8, length = 29)

Looked at the code, and the MKL implementation is actually the only implementation, so we probably don't need to say "if you need fast calculations" (who would prefer slow calculations anyway? :-)).

Comment (codemzs) [reply src/Microsoft.ML.StandardTrainers/Standard/LogisticRegression/LogisticRegression.cs]: lol

Comment (yaeldekel) [reply src/Microsoft.ML.StandardTrainers/Standard/LogisticRegression/LogisticRegression.cs]: So change to something like "To compute the standard deviation, use this..."

---
In reply to: [278327642](https://github.com/dotnet/machinelearning/pull/3572#discussion_r278327642) [](ancestors = 278327642)

Comment (codemzs) [reply src/Microsoft.ML.StandardTrainers/Standard/LogisticRegression/LogisticRegression.cs]: Changed to Use this class' implementation in the Microsoft.ML.Mkl.Components package which uses Intel Math Kernel Library....also the hardware acceleration part was a little dramatic

**Pull request merged**